database/factories/RatingFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Faker\Generator as Faker;
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+
+$factory->define(App\Models\Rating::class, function (Faker $faker) {
+    return [
+        'rating' => rand(1,5),
```

Comment:
```suggestion
        'rating' => rand(1, 5),
```
Sau dấu phẩy có dấu cách